_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+* Adding oauth2-proxy as optional alternative to oidc-authservice ([#2409](https://github.com/kubeflow/manifests/pull/2409))
+
+
+Future versions will continue to refine this rootless functionality, along with plans to make istio-cni the default option along with istio ambient mesh and incorporating [Pod Security Standards](https://github.com/kubeflow/kubeflow/pull/7325). Please join the [Security Working Group](https://kubeflow.slack.com/archives/C04CF10FJ2D)
+
+### Notebooks
+
+The v1.8 Notebook enhancements [(#7357)](https://github.com/kubeflow/kubeflow/pull/7357) includes support for ARM64 and a cleaner build system and Makefiles i.e. you now build any image locally by going to its folder and running make docker-build-dep to build with all base images that that image depends on. 1.8 also includes dependency updates to: Python 3.11.6, Tensorflow 2.13.0 and CUDA 11.8 for Tensoflow CUDA images, to Python 2.1.0 and CUDA 12.1 for Python CUDA images, and JupyterLab 4.0.
+
+
+### Platform dependencies, breaking changes, add-ons
+
+Kubeflow 1.8 includes hundreds of commits. The Kubeflow release process includes several rounds of testing by the Kubeflow working groups and Kubeflow distributions. Kubeflow’s configuration options provide a high degree of flexibility. After considering all of the testing options, the 1.8 Release Team narrowed the critical dependencies for consistent testing and documentation to the following.
+
+
+
+
+<table>
+  <tr>
+   <td><strong>Component</strong>
+   </td>
+   <td><a href="https://knative.dev/docs/reference/relnotes/">Knative</a>
+   </td>
+   <td><a href="https://istio.io/latest/news/releases/">Istio</a>
+   </td>
+   <td><a href="https://kubernetes.io/releases/">Kubernetes</a>
+   </td>
+   <td><a href="https://cert-manager.io/docs/installation/supported-releases/">Cert-Manager</a>
+   </td>
+   <td><a href="https://github.com/kubernetes-sigs/kustomize/releases">Kustomize</a>
+   </td>
+   <td><a href="https://github.com/dexidp/dex/releases">Dex</a>
+   </td>
+   <td><a href="https://github.com/argoproj/argo-workflows/releases">Argo</a>
+   </td>
+   <td>Tekton
+   </td>
+   <td>Oidc
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Version used in 1.8</strong>
+   </td>
+   <td>1.10.2 (Serving) 1.10.1 (Eventing)
+   </td>
+   <td>1.17.3
+   </td>
+   <td>1.25/1.26
+   </td>
+   <td>1.12.2
+   </td>
+   <td>5.0.3
+   </td>
+   <td>2.36.0
+   </td>
+   <td>3.3.10
+   </td>
+   <td>0.47.5
+   </td>
+   <td>e236439
+   </td>
+  </tr>
+</table>
+
+
+ \
+The 1.8 documentation includes overall installation instructions from the Manifest Working Group, and detailed feature reviews from each Kubeflow working group. Most of the working groups have broken their changelogs into subsections that highlight core features, UI enhancements, miscellaneous updates, bug fixes and breaking changes.
```

Comment:
I think we should only list this information once.

I propose we remove this section and add any words that we want to keep to the duplicate `### Dependency Versions (Manifests)` section (which is already at the bottom of the page).

```suggestion
```